docs/how-to-guides/backup-rook-ceph-volumes.md

```diff
@@ -0,0 +1,264 @@
+
+```bash
+kubectl apply -f stateful.yaml
+```
+
+Verify the application is running fine:
+
+```console
+$ kubectl -n demo-ns get pods
+NAME         READY   STATUS    RESTARTS   AGE
+demo-app-0   1/1     Running   0          16s
+```
+
+Execute the following command to generate some dummy data:
+
+```console
+kubectl -n demo-ns exec -it demo-app-0 -- /bin/sh -c \
+    'dd if=/dev/zero of=/data/file.txt count=40 bs=1048576'
+```
+
+Verify that the data is generated:
+
+```console
+$ kubectl -n demo-ns exec -it demo-app-0 -- /bin/sh -c 'du -s /data'
+40960   /data
+```
+
+### Step 3: Annotate pods
+
+Annotate the pods with volumes attached to them with their volume names. Replace the pod name and
+the volume name as needed in the following command:
+
+```bash
+kubectl -n demo-ns annotate pod demo-app-0 backup.velero.io/backup-volumes=data
+```
+
+### Step 4: Backup entire namespace
+
+Execute the following command to start the backup of the concerned namespace. In our demo
+application's case it is `demo-ns`:
+
+```bash
+velero backup create backup-demo-app-ns --include-namespaces demo-ns --wait
+```
+
+Above operation may take some time, depending on the size of the data.
+
+### Step 5: Restore Volumes
+
+#### Same Cluster
+
+If you plan to restore in the same cluster, then delete the namespace. In case of our demo
+application run the following command:
+
+```bash
+kubectl delete ns demo-ns
+```
+
+> **NOTE**: If you are restoring a stateful component of Lokomotive like `prometheus-operator`, then
+> delete the component namespace by running `kubectl delete ns monitoring`.
+
+#### Different Cluster
+
+In another cluster deploy components `rook`, `rook-ceph` and `velero` with the same configuration
+for a successful restore.
+
+#### Restore
+
+Execute the following command to start the restore:
+
+```bash
+velero restore create --from-backup backup-demo-app-ns
+```
+
+Verify if Velero restored the application successfully:
+
+```console
+$ kubectl -n demo-ns get pods
+NAME         READY   STATUS    RESTARTS   AGE
+demo-app-0   1/1     Running   0          51s
+```
+
+> **NOTE**: If you are restoring a stateful component of Lokomotive like `prometheus-operator`, then
+> once pods in `monitoring` namespace are in `Running` state, then run `lokoctl component apply
+> prometheus-operator` to ensure the latest configs are applied.
+
+Verify that the data is restored correctly:
+
+```console
+$ kubectl -n demo-ns exec -it demo-app-0 -- /bin/sh -c 'du -s /data'
+40960   /data
+```
+
+## Additional resources
+
+- Velero [Restic Docs](https://velero.io/docs/v1.4/restic/).
+- Lokomotive `velero` component [configuration reference
+  document](../configuration-reference/components/velero.md).
+- Lokomotive `rook` component [configuration reference
+  document](../configuration-reference/components/rook.md).
+- Lokomotive `rook-ceph` component [configuration reference
+  document](../configuration-reference/components/rook-ceph.md).
```

Comment:
Do we need line breaks here in the middle of label? That looks odd to me.

Reply:
It is exceeding line limit hence it is automatically wrapped around by my editor.